docs/DeveloperGuide.md

```diff
@@ -259,17 +264,21 @@ The remaining private methods represent separated cases for different commands.
 
 Shown below is a sequence diagram detailing how the Parser class interprets the user's CLI input
 and returns the appropriate command object containing the user's input parameters.
-![parseCommandShort.png](img/SequenceDiagrams/parseCommandShortSequenceDiagram.png)
+![parseCommandShort.png](img/SequenceDiagrams/ParseCommandShortSequenceDiagram.png)
+Note: The getXYZCommand(args) method is used for commands that further need to extract parameters from the user's input
+beyond just the command name, such as for the sort command which needs to extract from the input, the type of sort that
+is requested. In the example of the sort command, the getXYZCommand(args) method is called `getSortCommand()`.
 
 <div style="page-break-after: always;"></div>
 
 ### Storage Class
 The Storage class 
 - saves the User's added recipes to a .txt file
+  - This is done every time the recipes are modified in any way (such as added, deleted, sorted or edited)
   - The following Sequence diagram shows how the saveRecipes() method in the Storage class
-  saves all the created recipes into the .txt file
+    saves all the created recipes into the .txt file
```

Comment:
Will this be shown in the same line? Github shows that it's 2 lines but I can't tell if it's correct or not...

Reply:
It is 2 lines on the .md file (to not exceeed the max characters per line limit) but will show as 1 line on the final html version (because there isnt an extra blank line between those 2 lines).